### DISCUSSION TITLE
UI issue in mobile view

### DISCUSSION THREAD

**Discussion by ghost** [orestbida/cookieconsent#286] [Q&A] [answered]
Hi the text is out of view I test it with iphone5 and any similar size will face same issue. 
![Screenshot from 2022-04-03 14-45-14](https://user-images.githubusercontent.com/97788614/161428770-0315a18f-cf71-4735-80f3-906babfb1db4.png)

**Comment by orestbida** [comment, marked as answer]
Answered this here: #285.
